.github/workflows/version_bump.yml

```diff
@@ -27,6 +27,10 @@ jobs:
           with:
             node-version: 16
 
+        # Transpile the typescript files and minify javascript files
+        - name: Run minifyTs & minifyJs  
+          run: npm run production
```

Comment:
These changes are no where mentioned in commit message of PR description. @divyjainjd

Reply:
Added this changes in description @jainarpit014